subsystems/TypeHandlerLibrary/src/test/java/org/terasology/persistence/typeHandling/InMemorySerializerTest.java

```diff
@@ -38,8 +38,11 @@
 import java.util.stream.Stream;
 
 
+// spotbugs does thinks Assertions.assertThrows does not throw the exception, even if
+// test is successful. see bug: https://github.com/spotbugs/spotbugs/issues/2667.
+@edu.umd.cs.findbugs.annotations.SuppressFBWarnings
```

Comment:
seems to be fixed with https://github.com/spotbugs/spotbugs/pull/2648 but not released yet
we don't fail for spotbugs findings at the moment, so I would not suppress this now but wait for the spotbugs patch

Reply:
agree and took it out.

Reply:
new spotbug version now included, warnings are gone.